helper/gitcc/setup.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+from setuptools import find_packages, setup
+
+setup(
+    name="GIT Commit Check",
+    version="1.0.0",
+    description="Checks the commit name message",
+    long_description="",
+    long_description_content_type='text/x-rst',
+    author="Iceflower S",
+    author_email="iceflower@gmx.de",
+    license='MIT',
+    url="",
+    classifiers=[
+        'Programming Language :: Python :: 3.8',
+        'License :: OSI Approved :: MIT License',
+        'Development Status :: 5 - Production/Stable',
+        'Operating System :: OS Independent',
+        'Intended Audience :: Developers',
+        'Natural Language :: English',
+        'Environment :: Console',
+    ],
+    keywords='modular downloader',
```

Comment:
Copy pasta from another project...
```suggestion
    keywords='git commit naming check',
```